torch/csrc/jit/codegen/cuda/manager.cpp

```diff
@@ -52,6 +52,25 @@ namespace cuda {
 
 namespace {
 
+// Mark string attribute in alias-copy nodes to enable its implementation
+// in the fallback path.
+void enableAliasCopyNodes(const std::shared_ptr<Graph>& graph, Block* block) {
```

Comment:
> we don't need this any more. I think we can use `revertAliasCopyOps`. Similar refactor should be done infallback graph used by fusion guard. implementation of xxxx_copy ops should be removed.

nitpick, I'll do a clean up afterwards. Do you mind putting a TODO here and copy-paste

Reply:
Added the TODO